src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -18,15 +18,16 @@
 import java.awt.BorderLayout;
 
 import java.io.IOException;
-
+import java.lang.reflect.Array;
```

Comment:
Yes, was being dumb about array size.  Turns out there is a length member.  I have coded some Java a long time ago, but clearly still a newb :)